Eigen/src/SparseQR/SparseQR.h

```diff
@@ -600,16 +624,20 @@ void SparseQR<MatrixType,OrderingType>::factorize(const MatrixType& mat)
 template <typename SparseQRType, typename Derived>
 struct SparseQR_QProduct : ReturnByValue<SparseQR_QProduct<SparseQRType, Derived> >
 {
-  typedef typename SparseQRType::QRMatrixType MatrixType;
+  typedef typename SparseQRType::MatrixType MatrixType;
```

Comment:
@vkhristenko do we really need this change ?
it seems something that can have consequences outside CUDA.

Reply:
no, this is not needed. Must be a mistake of mine. Sparse matrices are not used. Were tested, but not used anywhere by me. That needs to be reverted.

Reply:
Done, thanks.